pkg/api/v1alpha1/condition_consts.go

```diff
@@ -0,0 +1,49 @@
+package v1alpha1
+
+// Conditions and condition Reasons for the Cluster object.
+const (
+	// ReadyCondition reports a summary of other conditions, indicating an overall operational
+	// state of the cluster.
+	ReadyCondition ConditionType = "Ready"
+
+	// PendingUpdateReason reports when an update has been scheduled but has yet to happen.
```

Comment:
This is a bit "obscure", like an "update" to what?
maybe something in the lines of "OutdatedInformationReason" "reports the system is waiting for stale cluster information to be refreshed"